src/js/index.js

```diff
@@ -171,15 +183,17 @@ const initPlugin = function(player, options) {
     if (!error) {
       return;
     }
+
     error = videojs.mergeOptions(error, options.errors[error.code || 0]);
+
     if (error.message) {
       details = `<div class="vjs-errors-details">${player.localize('Technical details')}
         : <div class="vjs-errors-message">${player.localize(error.message)}</div>
         </div>`;
     }
+
     if (error.code === 4 && FlashObj && !FlashObj.isSupported()) {
-      const flashMessage = player.localize(' * If you are using an older browser' +
-      ' please try upgrading or installing Flash.');
+      const flashMessage = player.localize('If you are using an older browser please try upgrading or installing Flash.');
```

Comment:
I don't think decorative elements such as this should be included in localizable strings.